update.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+# download the latest Prototype Kit zip file
+if [ ! -f govuk-prototype-kit.zip ]; then
+	curl -Lo govuk-prototype-kit.zip https://govuk-prototype-kit.herokuapp.com/docs/download
+fi
+
+# unzip the file into the 'update' folder
+if [ ! -d update ]; then
+	unzip -d update govuk-prototype-kit.zip
+fi
```

Comment:
can we move `update/govuk-prototype-kit-*` to `update`

Reply:
Unfortunately no, I think it messes with rsync (I can't get it to work). But we can get it to print out the folder, if that helps.

Reply:
maybe we can try again now we're not using rsync

Reply:
Grr, ran into another issue... we can't do the `echo '*' > update/.gitignore` hack if we're extracting the contents of the zip file into the update folder, because then the `.gitignore` file will be replaced by the one in the prototype kit. I guess we could fix this going forwards by making sure that we have the `update` folder in the `.gitignore` file for future prototype kit releases, but to help users now we'd have to make another release, which we're trying to avoid :/

Reply:
theoretically it shouldnt matter should it? If people delete the update folder as per the instructions

Reply:
It took half a day, but I managed it. Files are now extracted to the update folder.